src/components/Table/Table.jsx

```diff
@@ -136,10 +139,17 @@ class Table extends React.Component {
     return this.renderHeader({ sticky: true });
   }
 
+  renderCarat() {
+    return this.props.sort.order === 'asc' ?
+      <Icon color="#fff" icon={downArrow} />
+      : <Icon color="#fff" icon={upArrow} />;
```

Comment:
maybe combine? 

`return <Icon color="#fff" icon={this.props.sort.order === 'asc' ? downArrow : upArrow} />`